inlong-sort/sort-connectors/jdbc/src/main/java/org/apache/inlong/sort/jdbc/table/JdbcDynamicTableFactory.java

```diff
@@ -219,6 +221,7 @@ public DynamicTableSource createDynamicTableSource(Context context) {
 
     private JdbcOptions getJdbcOptions(ReadableConfig readableConfig) {
         final String url = readableConfig.get(URL);
+        JdbcUrlUtils.replaceInvalidUrlProperty(url);
```

Comment:
`JdbcUrlUtils.replaceInvalidUrlProperty(url);` return value should be used.

Reply:
noted thx